src/main/scala/org/ergoplatform/modifiers/history/BlockTransactions.scala

```diff
@@ -35,10 +38,17 @@ case class BlockTransactions(headerId: ModifierId,
 
   lazy val txIds: Seq[Array[Byte]] = txs.map(_.serializedId)
 
+  //todo: add extra byte to separate from txIds?
+  lazy val witnessIds: Seq[Array[Byte]] = txs.map(_.witnessSerializedId)
```

Comment:
Please add ScalaDocs for public methods.

Reply:
done